R/zzz.R

```diff
@@ -86,6 +87,18 @@
 	}
 }
 
+.modify <- function(object) {
+  if (is(object, "CsvFileTable")){
+    # reading from csv
+    unlockBinding("asDataFrame", object)
+    object$asDataFrame <- function() {
+      read.csv(object$filepath, encoding="UTF-8", stringsAsFactors=FALSE, check.names=FALSE, na.strings=c(""))
```

Comment:
please break this out as readCsv() and then add a test for the writeCsv() -> readCsv() round trip.